backend/graph/api/schema/sections.graphqls

```diff
@@ -7,7 +7,7 @@ type ItemSectionMetadata {
     prependLiveElement: Boolean!
 }
 
-interface Section{
+interface Section implements CollectionItem {
```

Comment:
Sections doesnt make sense as collection items

Reply:
Oh true. It's supported by the backend, though. Just not exposed anywhere (not selectable in admin either). 

Probably a remnant from an older feature.

Reply:
I can remove it if it's something we don't want to have at all.

Reply:
yea probably good to remove it, it would help keep things clean/understandable 👍